src/pages/Operations/index.tsx

```diff
@@ -93,37 +91,29 @@ const Operations: React.FC<OperationsProps> = ({
     }
   }, [balances, asset_id, network]);
 
-  const openTxRow = (txID: string) => setTxRowOpened([...txRowOpened, txID]);
-
-  const closeTxRow = (txID: string) => setTxRowOpened(txRowOpened.filter((id) => id !== txID));
-
-  const isTxRowOpen = (tx: TxDisplayInterface): boolean => {
-    if (TxTypeEnum[tx.type] === 'DepositBtc') {
-      const isClaimed = !!tx.claimTxId;
-      const isClaimable = checkIfPeginIsClaimable(tx);
-      if (!isClaimed && !isClaimable) {
-        // Not claimed, not claimable => closed by default, normal onClick behavior
-        return txRowOpened.includes(tx.txId);
-      } else if (!isClaimed && isClaimable) {
-        // Not claimed, claimable => open and button to claim
-        return true;
-      } else if (isClaimed) {
-        // Claimed => closed by default, normal onClick behavior
-        return txRowOpened.includes(tx.txId);
-      } else {
-        return txRowOpened.includes(tx.txId);
-      }
-    } else {
-      return txRowOpened.includes(tx.txId);
-    }
-  };
-
   const onclickTx = (tx: TxDisplayInterface) => {
```

Comment:
what happens on pegin now?

Reply:
I think that expanding behavior for pegins should remain

Reply:
Everything is expanded by default, clicking on it does nothing